gitops/generate_build.go

```diff
@@ -172,7 +173,7 @@ func DetermineBuildExecution(component appstudiov1alpha1.Component, params []tek
 			},
 		)
 		// imagePullSecret to be set in the podTemplate used by chains
-		pipelineRunSpec.PodTemplate = &tektonapi.PodTemplate{
+		pipelineRunSpec.PodTemplate = &pod.PodTemplate{
```

Comment:
any reason why we are switching to a different api here?

Reply:
yes, this is a result of updating the `github.com/tektoncd/pipeline` package.  They refactored the pod template in v0.39.0 https://github.com/tektoncd/pipeline/commit/8413f854c11723ce2e0917284e4424ad4d4a9b5b